docs/Installing-Clear-Containers-3.0-on-Fedora.md

```diff
@@ -0,0 +1,54 @@
+# Installing Clear Containers 3.0 Alpha on Fedora
+
+## Introduction
+
+Clear Containers **3.0-alpha** is available for Fedora\* versions **24** and **25**.
+
+## Install latest Docker
+
+This step is only required in case Docker is not installed on the system.
+
+```
+$ sudo dnf -y install dnf-plugins-core
+$ sudo dnf config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo
+$ sudo dnf makecache fast
+$ sudo dnf install docker-ce
+```
+
+For more information on installing Docker please refer to the
+[Docker Guide](https://docs.docker.com/engine/installation/linux/fedora)
+
+## Install Clear Containers 3.0 components
```

Comment:
Do not use headings for steps. Please use ordered lists:
2. Install the Clear Containers 3.0 components with the following commands: